pageserver/src/consumption_metrics.rs

```diff
@@ -110,7 +110,12 @@ pub async fn collect_metrics_iteration(
     );
 
     // get list of tenants
-    let tenants = mgr::list_tenants().await;
+    let tenants = match mgr::list_tenants().await {
+        Ok(tenants) => tenants,
+        Err(e) => {
+            anyhow::bail!(e)
+        }
+    };
```

Comment:
```suggestion
    let tenants = match mgr::list_tenants().await?;
```

Cannot at least see why this wouldn't work.